pkg/util/common/util.go

```diff
@@ -197,3 +198,13 @@ func TimestampMillisToTime(ts uint64) time.Time {
 	ns := ts64 % 1000 * 1000000
 	return time.Unix(s, ns)
 }
+
+// Replaces invalid utf8 characters with '?'.
+func ReplaceInvalidUtf8Chars(s string) string {
+	return strings.Map(func(r rune) rune {
+		if r == utf8.RuneError {
+			return '?'
```

Comment:
Should be replaced with special replacement character as in https://play.golang.org/p/4NnEknJtD9q

Reply:
More test cases:

TX ID | Name  B64 | Description B64
------------ | -------------|-------------
4NWVZHR3pmtmUaMACCFPQnV5Bebcxt1PdoKeRUVQWBwu|u3UA7U/SvRr76A==|SnLxCohBOyY3+g==
AnWV1Jg3JiWZjPYGta37i8CyHayrbpQrAogNf3Es4UhE|kzS+SlILWaCsyg==|CMWs9AG50/8lTg==
DTEutun27anDjdy2D4kWCXPv7uU84g2LBVw3cgCxMeJm|BWFwzuJoSjAwrA==|cS5ckfL2T75itQ==
weL4JMgoADYkjW2vsY6bZ6iEc6AYtabWEMsopEBpezi|no7KxsdNrk+zGg==|WhD3HJkJlEaiwg==
4Zm1SCDabGK7qPpXBxTTWD9DuW6wb7juSR9zvdXra4LZ|H5PBVxrwsvtp5A==|M4LcsQ0uPXSpFw==
B4JeyxHvMtxdTuqi8QzprwM4TAXZEj9f4GjqQDGT6x86|As7ayhU0UVXXiQ==|MAfv7G4JcPhobg==
GPdqzxFkDMsjvTVzLvBm3VhBYjYTPhUgcw2Xr5Xc1gz6|lNorML+XJ2QInw==|bFBv78zAZtvOIg==